_includes/description.md

```diff
@@ -8,7 +8,7 @@ As you can see the `description` field is set to be `test purchase -
 orderNumber28749347` in the the code example. The images below will show you the
 payment UI for the Redirect and Seamless View scenario.
 
-{% include alert.html type="neutral" icon="info" body="Notice that for Redirect,
+{% include alert.html type="informative" icon="info" body="Notice that for Redirect,
```

Comment:
Is this the only `neutral` alert we have? If you haven't, can you do a quick search and replace?